CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 - Fix handling of `TUIST_CONFIG_COLOURED_OUTPUT` environment variable [#3631](https://github.com/tuist/tuist/pull/3631) by [@danyf90](https://github.com/danyf90)
 - Fix `tuist dump config` no longer requires to be executed inside the `Tuist` folder [#3647](https://github.com/tuist/tuist/pull/3647) by [@danyf90](https://github.com/danyf90)
+- Calculation of the macOS minimum deployment target for Swift Packages that don't specify it [#3639](https://github.com/tuist/tuist/pull/3639) by [@pepibumur](https://github.com/pepibumur)
```

Comment:
I think we don't need a changelog entry, as we already have the one introducing the "feature" (the first in Changes)